identify/extensions.py

```diff
@@ -183,8 +183,10 @@
     'r': {'text', 'r'},
     'rake': {'text', 'ruby'},
     'rb': {'text', 'ruby'},
+    'rdf': {'text', 'xml', 'rdf-xml'},
```

Comment:
idk that the `-xml` adds anything here -- unless there's a reason to have a distinction many of the tags just match their extensions

Reply:
Thank you for you review! It's to follow the common naming of the format: https://en.wikipedia.org/wiki/RDF/XML

Reply:
is there a non-xml rdf?

Reply:
Yes, [Turtle](https://www.w3.org/TR/2014/REC-turtle-20140225/) (`.ttl`, a plain text format), [N-Triples](https://www.w3.org/TR/2014/REC-n-triples-20140225/) (`.nt`, a CSV-like format) and [JSON-LD](https://www.w3.org/TR/json-ld/) (`.json-ld`, already in the list, a JSON-based format).